### PR TITLE
[AMSDK-10380] Griffon -> Assurance

### PR DESCRIPTION
**Pull request by nporter-adbe**
Replaces Griffon usage with Assurance.

### CODE REVIEW

File: AEPExperiencePlatform.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1501,6 +1507,8 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = com.adobe.mobile.AEPCommerceDemoApp;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_OBJC_BRIDGING_HEADER = "SampleApps/AEPCommerceDemoApp/AEPCommerceDemoApp/AEPCommerceDemoApp-Bridging-Header.h";
```
Comment (emdobrin) [inline AEPExperiencePlatform.xcodeproj/project.pbxproj:1510]: This should not be required anymore

File: SampleApps/AEPCommerceDemoApp/AEPCommerceDemoApp/AEPDemoConstants.swift [hunk 2]
```diff
@@ -46,15 +46,15 @@ struct AEPDemoConstants {
...
-        static let griffonSessionDisconnected = "Griffon Session has been disconnected"
-        static let griffonSessionNotActive = "Griffon Session is not active..."
-        static let griffonUrlValidationString = "adb_validation_sessionid"
+        // MARK: Assurance : Labels of UI Components
+        static let assuranceConnect = "Connect"
+        static let assuranceDisconnect = "Disconnect"
+
+        // MARK: Assurance : Snackbar messages
+        static let assuranceUrlInvalid = "Enter a valid URL of the Assurance Session"
+        static let assuranceSessonActive = "Connection to a Assurance Session is already active or in progress. Disconnect before trying to connect..."
```
Comment (emdobrin) [inline SampleApps/AEPCommerceDemoApp/AEPCommerceDemoApp/AEPDemoConstants.swift:55]: Can we reword this one to: `A connection to an Assurance Session is already ...`

File: SampleApps/AEPCommerceDemoApp/AEPCommerceDemoApp/AppDelegate.swift [hunk 3]
```diff
@@ -29,9 +30,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         MobileCore.setLogLevel(level: .trace)
         MobileCore.configureWith(appId: LAUNCH_ENVIRONMENT_FILE_ID)
 
-        // todo: reference the swift griffon extension here
-        //ACPGriffon.registerExtension()
-        MobileCore.registerExtensions([Identity.self, Lifecycle.self, ExperiencePlatform.self])
+        AEPAssurance.registerExtension()
+        ACPCore.registerExtensions([Identity.self, Lifecycle.self, ExperiencePlatform.self])
+        // MobileCore.updateConfigurationWith(configDict: config)
```
Comment (emdobrin) [inline SampleApps/AEPCommerceDemoApp/AEPCommerceDemoApp/AppDelegate.swift:35]: Let's remove this if not used.

File: SampleApps/AEPCommerceDemoApp/AEPCommerceDemoApp/AEPDemoConstants.swift [hunk 4]
```diff
@@ -46,15 +46,15 @@ struct AEPDemoConstants {
...
+        static let assuranceSessionDisconnected = "Assurance Session has been disconnected"
```
Comment (emdobrin) [inline SampleApps/AEPCommerceDemoApp/AEPCommerceDemoApp/AEPDemoConstants.swift:56]: I think this is not used anymore, let's clean up.

File: Sources/ExperiencePlatform.swift [hunk 5]
```diff
@@ -94,11 +94,11 @@ public class ExperiencePlatform: NSObject, Extension {
             Log.warning(label: LOG_TAG, "handleExperienceEventRequest - An unexpected error has occurred, ECID is nil.")
         }
 
-        // get Griffon integration id and include it in to the requestHeaders
+        // get Assurance integration id and include it in to the requestHeaders
         var requestHeaders: [String: String] = [:]
-        if let griffonSharedState = getSharedState(extensionName: Constants.SharedState.Griffon.STATE_OWNER_NAME, event: event)?.value {
-            if let griffonIntegrationId = griffonSharedState[Constants.SharedState.Griffon.INTEGRATION_ID] as? String {
-                requestHeaders[Constants.NetworkKeys.HEADER_KEY_AEP_VALIDATION_TOKEN] = griffonIntegrationId
+        if let assuranceSharedState = getSharedState(extensionName: Constants.SharedState.STATE_OWNER, event: event)?.value {
```
Comment (emdobrin) [inline Sources/ExperiencePlatform.swift:99]: getSharedState(extensionName: Constants.SharedState. Assurance.STATE_OWNER, event: event)?.value

**Review by emdobrin — CHANGES_REQUESTED** [review]

File: AEPExperiencePlatform.xcodeproj/project.pbxproj (hunk 1 above)
Comment (nporter-adbe) [reply AEPExperiencePlatform.xcodeproj/project.pbxproj:1510]: I attempted to remove it, however, importing `AEPAssurance` still fails.

File: Sources/ExperiencePlatform.swift (hunk 5 above)
Comment (nporter-adbe) [reply Sources/ExperiencePlatform.swift:99]: thanks, looks like this got messed up when handling merge conflicts.

File: AEPExperiencePlatform.xcodeproj/project.pbxproj (hunk 1 above)
Comment (nporter-adbe) [reply AEPExperiencePlatform.xcodeproj/project.pbxproj]: nevermind, had to remove all the pods and re-add. working now.

File: SampleApps/AEPCommerceDemoApp/AEPCommerceDemoApp/Info.plist [hunk 6]
```diff
@@ -16,6 +16,19 @@
...
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>Griffon Deeplink</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>griffon</string>
```
Comment (emdobrin) [inline SampleApps/AEPCommerceDemoApp/AEPCommerceDemoApp/Info.plist:28]: Let's use commerceapp or something similar instead

**Review by emdobrin — APPROVED** [review]
Looks good, left one small comment

**Pull request merged**